src/components/BookingForm.vue

```diff
@@ -36,8 +34,10 @@
 </template>
 
 <script>
+import BadDatePicker from './BadDatePicker.vue';
```

Comment:
You don't need to import the component, just use it.

Reply:
After removing this, the tests will probably fail. Take a look at the BookingForm.spec.js to see how I'm using the findComponent method.

Reply:
I will take a look, thanks!